_rules/table-header-cells-refer-to-data-cells-d0f69e.md

```diff
@@ -0,0 +1,247 @@
+		</div>
+	</div>
+</div>
+```
+
+#### Passed Example 3
+
+Each column header element has assigned cells within the same table element. In this example the column headers have cells that span multiple columns.
+
+```html
+<table>
+	<thead>
+		<tr>
+			<th>Projects</th>
+			<th>Exams</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td colspan="2">15%</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+#### Passed Example 4
+
+Each row and column header element has assigned cells, within the same element having a [semantic role](#semantic-role) of `grid`.
+
+```html
+<table role="grid">
+	<thead>
+		<tr role="row">
+			<td></td>
+			<th role="columnheader">Breakfast</th>
+			<th role="columnheader">Lunch</th>
+			<th role="columnheader">Dinner</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr role="row">
+			<th scope="row" role="rowheader">Day 1</th>
+			<td>8:00</td>
+			<td>13:00</td>
+			<td>18:00</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+#### Passed Example 5
+
+Each column header is assigned to a cell. Usage of `headers` attribute changes the relationship between column headers and cells.
+
+```html
+<table>
+	<tr>
+		<th id="col1">Column 1</th>
+		<th id="col2">Column 2</th>
+	</tr>
+	<tr>
+		<td></td>
+	</tr>
+	<tr>
+		<td headers="col2"></td>
+	</tr>
+</table>
+```
+
+### Failed
+
+#### Failed Example 1
+
+One of the column headers ("Column 2"), does not have an assigned cell within the same `table` element.
+
+```html
+<table>
+	<thead>
+		<tr>
+			<th>Column 1</th>
+			<th>Column 2</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td>15%</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+#### Failed Example 2
+
+The second column header, does not have an assigned cell within the same `table` element. Usage of `headers` attribute removes cell association to the column.
+
+```html
+<table>
+	<tr>
+		<th id="col1"></th>
+		<th id="Column2"></th>
+	</tr>
+	<tr>
+		<td></td>
+		<td headers="col1"></td>
+	</tr>
+</table>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The rule does not apply to table element that is not [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+```html
+<table role="presentation">
+	<tr>
+		<th>Time</th>
+	</tr>
+	<tr>
+		<td>12:00</td>
+	</tr>
+</table>
+```
+
+#### Inapplicable Example 2
+
+The rule does not apply to table element that is not [visible](#visible) in page.
+
+```html
+<html>
+	<style>
+		.notInPage {
+			position: absolute;
+			left: -9999px;
+			top: -9999px;
+		}
+	</style>
+	<body>
+		<table class="notInPage">
+			<tr>
+				<th>Time</th>
+			</tr>
+			<tr>
+				<td>24:00</td>
+			</tr>
+		</table>
+	</body>
+</html>
+```
+
+#### Inapplicable Example 3
+
+The rule does not apply to table element that has no [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) with [semantic roles](#semantic-role) of either [rowheader](https://www.w3.org/TR/wai-aria-1.1/#rowheader) or [columnheader](https://www.w3.org/TR/wai-aria-1.1/#columnheader).
+
+```html
+<table role="presentation">
```

Comment:
Seems unnecessary. Just a simpler version of inapplicable 1.

Reply:
Not really, but not perfect either 😉 
Inapplicable Example 1 has a `th` and is inapplicable because of `role="presentation"`.
This one does not have any `th` and is inapplicable because of this. The `role="presentation"` should be removed from that example.